lib/mbta_v3_api/store.ex

```diff
@@ -27,4 +27,9 @@ defmodule MBTAV3API.Store do
   Retrieve all records that match the given filter keys.
   """
   @callback fetch(keyword()) :: [JsonApi.Object.t()]
+
+  @doc """
+  Retrieve all records that match any of the given filter keyword sets
+  """
+  @callback fetch_multi_filter([keyword()]) :: [JsonApi.Object.t()]
```

Comment:
I think `fetch_any/1` might be a better name than `fetch_multi_filter/1`. Technically, it looks like it'd be possible to just write an ETS query with `or` in `fetch/1` and drop this function entirely, but that may be more work to implement for no particular benefit.

Reply:
👍 will rename! I initially intended to have a single `fetch` with pattern matching to determine the behavior depending on if the param was a list of keywords or a keyword, but there isn't an `is_keyword` guard. It ended up seeming a bit clearer to me to just have a distinctly named function

Reply:
Ah, there's no `is_keyword` guard, but you could `when is_tuple(hd(keyword_or_list))` if you wanted a similar effect - depending on whether an empty list should match everything or nothing, that may not quite be enough, though.

Reply:
Ooh there is a `Keyword.keyword?/1` function which isn't quite as good as a guard but seems a bit more readable. I ended up rolling some of this change forward into the next PR